src/interfaces/extensionSettings.ts

```diff
@@ -2,4 +2,5 @@ export interface ExtensionSettings {
   ansible: { path: string; useFullyQualifiedCollectionNames: boolean };
   ansibleLint: { enabled: boolean; path: string; arguments: string };
   python: { interpreterPath: string; activationScript: string };
+  ansiblePlaybook: { path: string };
```

Comment:
In [a similar situation](https://github.com/ansible/ansible-language-server/blob/main/src/services/ansibleConfig.ts#L30) , instead of an additional configuration item for users to fill in, I've implemented a [nice workaround/ugly hack] and re-used the value that was already provided for the `ansible` executable. Since all ansible executables are always installed in one place, this should be no problem.

Reply:
Thanks for suggesting this change. Makes sense!